demos/python_demos/requirements.txt

```diff
@@ -1,5 +1,10 @@
+matplotlib
+motmetrics
 nibabel
 numpy
 scikit-learn
 scipy
+setuptools
```

Comment:
It doesn't make sense to put `setuptools` here, because you need `setuptools` to use a requirements file to begin with.

Reply:
Removed.